vic/drivers/shared_all/src/history_metadata.c

```diff
@@ -0,0 +1,1482 @@
+
+    /* total net evaporation [mm] */
+    strcpy(out_metadata[OUT_EVAP].varname, "OUT_EVAP");
+    strcpy(out_metadata[OUT_EVAP].long_name, "evap");
+    strcpy(out_metadata[OUT_EVAP].standard_name, "water_evaporation_flux_net");
+    strcpy(out_metadata[OUT_EVAP].units, "mm");
+    strcpy(out_metadata[OUT_EVAP].description, "total net evaporation");
+
+    /* net evaporation from bare soil [mm] */
+    strcpy(out_metadata[OUT_EVAP_BARE].varname, "OUT_EVAP_BARE");
+    strcpy(out_metadata[OUT_EVAP_BARE].long_name, "evap_bare");
+    strcpy(out_metadata[OUT_EVAP_BARE].standard_name,
+           "water_evaporation_from_soil");
+    strcpy(out_metadata[OUT_EVAP_BARE].units, "mm");
+    strcpy(out_metadata[OUT_EVAP_BARE].description,
+           "net evaporation from bare soil");
+
+    /* net evaporation from canopy interception [mm] */
+    strcpy(out_metadata[OUT_EVAP_CANOP].varname, "OUT_EVAP_CANOP");
+    strcpy(out_metadata[OUT_EVAP_CANOP].long_name, "evap_canop");
+    strcpy(out_metadata[OUT_EVAP_CANOP].standard_name,
+           "water_evaporation_from_canopy");
+    strcpy(out_metadata[OUT_EVAP_CANOP].units, "mm");
+    strcpy(out_metadata[OUT_EVAP_CANOP].description,
+           "net evaporation from canopy interception");
+
+    /* moisture that reaches top of soil column [mm] */
+    strcpy(out_metadata[OUT_INFLOW].varname, "OUT_INFLOW");
+    strcpy(out_metadata[OUT_INFLOW].long_name, "inflow");
+    strcpy(out_metadata[OUT_INFLOW].standard_name, "unset");
+    strcpy(out_metadata[OUT_INFLOW].units, "unset");
+    strcpy(out_metadata[OUT_INFLOW].description, "something");
+
+    /* incoming baseflow from lake catchment [mm] */
+    strcpy(out_metadata[OUT_LAKE_BF_IN].varname, "OUT_LAKE_BF_IN");
+    strcpy(out_metadata[OUT_LAKE_BF_IN].long_name, "lake_bf_in");
+    strcpy(out_metadata[OUT_LAKE_BF_IN].standard_name, "infiltration_amount");
+    strcpy(out_metadata[OUT_LAKE_BF_IN].units, "mm");
+    strcpy(out_metadata[OUT_LAKE_BF_IN].description,
+           "oisture that reaches top of soil column");
+
+    /* incoming volumetric baseflow from lake catchment [m3] */
+    strcpy(out_metadata[OUT_LAKE_BF_IN_V].varname, "OUT_LAKE_BF_IN_V");
+    strcpy(out_metadata[OUT_LAKE_BF_IN_V].long_name, "lake_bf_in_v");
+    strcpy(out_metadata[OUT_LAKE_BF_IN_V].standard_name,
+           "baseflow_volume_from_lake_catchment");
+    strcpy(out_metadata[OUT_LAKE_BF_IN_V].units, "m3");
+    strcpy(out_metadata[OUT_LAKE_BF_IN_V].description,
+           "incoming volumetric baseflow from lake catchment");
+
+    /* outgoing baseflow lake [mm] */
+    strcpy(out_metadata[OUT_LAKE_BF_OUT].varname, "OUT_LAKE_BF_OUT");
+    strcpy(out_metadata[OUT_LAKE_BF_OUT].long_name, "lake_bf_out");
+    strcpy(out_metadata[OUT_LAKE_BF_OUT].standard_name,
+           "baseflow_outgoing_lake");
+    strcpy(out_metadata[OUT_LAKE_BF_OUT].units, "mm");
+    strcpy(out_metadata[OUT_LAKE_BF_OUT].description,
+           "somethoutgoing baseflow lakeing");
```

Comment:
not sure what this description is supposed to be, "outgoing baseflow from lake" perhaps?